target/linux/ramips/image/mt7621.mk

```diff
@@ -699,6 +699,16 @@ define Device/gnubee_gb-pc2
 endef
 TARGET_DEVICES += gnubee_gb-pc2
 
+define Device/hanyang_hyc-g920
+  $(Device/dsa-migration)
```

Comment:
Considering it's a new device I wonder if this can be skipped... 
But if this gets backported than it would makes sense

Reply:
I still don't understand well but Adrian once told me that it should not be skipped even for new devices.
https://github.com/openwrt/openwrt/pull/4416#discussion_r691544161